components/energylevel.html

```diff
@@ -0,0 +1,52 @@
+<h2>Energy Level throughout the day</h2>
+<section class="energylevel">
+  <ul class="energylevel__percentage">
+    <li>100</li>
+    <li>75</li>
+    <li>50</li>
+    <li>25</li>
+    <li>0</li>
+  </ul>
+
+  <div class="energylevel__bars energylevel__background">
+    <div class="energylevel__bars-double-1">
+      <div class="energylevel__bars-orange-1"></div>
+      <div class="energylevel__bars-blue-1"></div>
+    </div>
+    <div class="energylevel__bars-double-2">
+      <div class="energylevel__bars-orange-2"></div>
+      <div class="energylevel__bars-blue-2"></div>
+    </div>
+    <div class="energylevel__bars-double-3">
+      <div class="energylevel__bars-orange-3"></div>
+      <div class="energylevel__bars-blue-3"></div>
+    </div>
+    <div class="energylevel__bars-double-4">
+      <div class="energylevel__bars-orange-4"></div>
+      <div class="energylevel__bars-blue-4"></div>
+    </div>
+    <div class="energylevel__bars-double-5">
+      <div class="energylevel__bars-orange-5"></div>
+      <div class="energylevel__bars-blue-5"></div>
+    </div>
+  </div>
+
+  <ul class="energylevel__time">
+    <li>10:00</li>
+    <li>15:30</li>
+    <li>16:15</li>
+    <li>18:25</li>
+    <li>19:00</li>
+  </ul>
+</section>
+<section class="buttons__positioning margin-t10 margin-b30">
+  <span class="buttons__energylevel-blue">Other Students</span>
+  <span class="buttons__energylevel-orange">You</span>
```

Comment:
Class-Benennung evtl. irreführend, da eigentlich kein Button. 
```suggestion
<section class="legend margin-t10 margin-b30">
  <span class="legend__energylevel-blue">Other Students</span>
  <span class="legend__energylevel-orange">You</span>
```

Reply:
Gute Anmerkung - hatten wir so übernommen, weil es ja an sich das gleiche Styling ist!!

Reply:
Very good point. Maybe:
```suggestion
<section class="energylevel__legend margin-t10 margin-b30">
  <span class="energylevel__legendbutton--blue">Other Students</span>
  <span class="energylevel__legendbutton--orange">You</span>
```